### ISSUE TITLE
more domains

### ISSUE THREAD

**Issue by giannigrd** [disposable-email-domains/disposable-email-domains#158]
Disposal domanis
------------------------
ada-duit.ga
aesopsfables.net
blangbling784yy.tk
cursodemicropigmentacao.us
dgnghjr5ghjr4h.cf
email2an.ga
hjgh545rghf5thfg.gq
khan007.cf
king-yaseen.cf
kjhjgyht6ghghngh.ml
mailserver2.cf
mailserver2.ga
mailserver2.ml
mailserver2.tk
mial.cf
mial.tk
mysafe.ml
mytools-ipkzone.gq
ralree.com
reksareksy78oy.ml
roewe.cf
roewe.ga
roewe.gq
roewe.ml
rpaowpro3l5ha.tk
ryumail.net
s.ea.vu
safer.gq
v-mail.xyz
vw-eos.cf
vw-eos.ga
vw-eos.gq
vw-eos.ml
vw-eos.tk
yhjgh65hghgfj.tk
yughfdjg67ff.ga
cool.fr.nf
courriel.fr.nf
jetable.fr.nf
moncourrier.fr.nf
monemail.fr.nf
monmail.fr.nf
PutThisInYourSpamDatabase.com
SendSpamHere.com
SpamHereLots.com
SpamHerePlease.com
TempEMail.net
www.e4ward.com
www.mailinator.com
emailfake.usa.cc
6.emailfake.ml
fake-email.pp.ua
tempmail.xxi2.com
spam1.0-00.usa.cc
163.com
yopmail.pp.ua
3.emailfreedom.ml
kloap.com
add3000.pp.ua
kmhow.com
one.fackme.gq
2.emailfake.ml
5.emailfake.ml
4.emailfake.ml
3.emailfake.ml
1.emailfake.ml
emailfake.nut.cc
ce.mintemail.com
emailtemporanea.net
1.justemail.ml
2.justemail.ml
3.justemail.ml
4.justemail.ml
5.justemail.ml
6.justemail.ml
10minutemail.davidxia.com
temp-mail.pp.ua
ass.pp.ua
a.gmail.gr.com
1.fackme.gq
2.fackme.gq
3.fackme.gq
4.fackme.gq
5.fackme.gq
6.fackme.gq
7.fackme.gq
8.fackme.gq
9.fackme.gq
7mail7.com
5.emailfreedom.ml
yhg.biz
youpmail.com
4.emailfreedom.ml
zasod.com
euaqa.com
ip6.pp.ua
mox.pp.ua
get.pp.ua
cdnqa.com
iaoss.com
ggg.pp.ua
foxja.com
pooae.com
figjs.com
haqed.com
---------------------------------
Invalid Domains : 

moditstudio.it
dm.w3internet.co.ukexample.com
h.mintemail.com
eladis.it
solartotal.com
solartotal.be
solartotal.de
solartotal.es
dsiay.com
3.kadokawa.top
af.net
al73.com
yurakucommunity.com
333solar.com
xvx.us
send-email.ml
noicd.com

**Comment by martenson** [comment]
@giannigrd can you tell me more about the source of these domain please?

**Comment by giannigrd** [comment]
I Martin,

from the my managed website, i extracted the list of e-mail domains used by customers registtrations

Check regularly if domains are valid through customers actions after sending activation mail
I check the anomalous situations with https://check-mail.org or similar sites


Best regards

Gianni Gardella









________________________________
Da: Martin Cech <notifications@github.com>
Inviato: lunedì 23 aprile 2018 17:00
A: martenson/disposable-email-domains
Cc: giannigrd; Mention
Oggetto: Re: [martenson/disposable-email-domains] more domains (#158)


@giannigrd<https://github.com/giannigrd> can you tell me more about the source of these domain please?

**Comment by slawa-dev** [comment]
@martenson You added those. You should close this issue.

**Comment by martenson** [comment]
@slawa-dev I don't believe we did, some are already there from different sources but the list above needs cleaning and a PR for the rest.

**Comment by martenson** [comment]
sorry it got so outdated, feel free to clean up and make a PR though
